disputer/index.js

```diff
@@ -57,15 +58,20 @@ async function run(price, address, shouldPoll) {
 
 const Poll = async function(callback) {
   try {
-    if (!argv.address) {
+    if (!process.env.EMP_ADDRESS) {
       throw new Error("Bad input arg! Specify an `address` for the location of the expiring Multi Party.");
```

Comment:
You should change the language here to reflect that its an environment variable not a flag